src/components/calcite-slider/calcite-slider.e2e.ts

```diff
@@ -148,7 +148,7 @@ describe("calcite-slider", () => {
     expect(ticks.length).toBe(11);
   });
 
-  it("fires calciteSliderChange event on changes", async () => {
+  it("keyboard interaction", async () => {
```

Comment:
nitpick: the title "keyboard interaction" sounds more appropriate for a `describe("keyboard interaction")` section in the tests.  Could you rename this test to make it more meaningful and if it is helpful to group all the keyboard tests together, wrap them in a `describe("keyboard interaction")`.